usr/share/rear/lib/layout-functions.sh

```diff
@@ -548,10 +550,16 @@ get_device_mapping() {
 }
 
 # Get the size in bytes of a disk/partition.
+# For disks, use "sda" as argument.
 # For partitions, use "sda/sda1" as argument.
 get_disk_size() {
     local disk_name=$1
 
+    if has_binary blockdev; then
+        blockdev --getsize64 /dev/${disk_name##*/}  # sda/sda1 -> sda1 ; sda -> sda
+        return 0
```

Comment:
This `return 0` is logically wrong because it returns 0
also when the `blockdev` command failed
i.e. it returns 0 in any case when a `blockdev` command exists.